eng/pipelines/pull_request.yml

```diff
@@ -70,21 +70,52 @@ parameters:
   displayName: Run Mono tests on Mac
   type: boolean
   default: true
+- name: isOfficialBuild
+  type: boolean
+  default: false
```

Comment:
Like a comment on another file, we don't need the ability to set `isOfficialBuild` to true when manually queueing the build. Therefore we don't need this parameter on this yaml.

Reply:
I unresolved this conversation, as I still see `isOfficialBuild` as a parameter that can changed on manual queue. Although, I don't see it actually passed through to the `pipeline.yml` template.